sdk/core/core-tracing/package.json

```diff
@@ -43,7 +43,12 @@
     "LICENSE"
   ],
   "repository": "github:Azure/azure-sdk-for-js",
-  "keywords": ["azure", "tracing", "Azure", "cloud"],
+  "keywords": [
```

Comment:
Autoupdated by npm. All others azure-sdk-for-js package.json files use multiline keywords.